packages/viewer/src/viewport-machine.ts

```diff
@@ -54,7 +58,7 @@ export const viewportMachine = createMachine({
                 context.cameraSubscription.unsubscribe();
               context.cameraSubscription = context.camera?.subscribe(
                 (state) => {
-                  self.send({
+                  (self as ActorRef<SetCameraEvent | SetCameraPoseEvent>).send({
```

Comment:
Remote `SetCameraPoseEvent`?

Reply:
I remember now what `SetCameraPoseEvent` does and added this comment:

Let observers of Viewport know that camera has updated